src/RoadRunner/RoadRunnerClient.php

```diff
@@ -47,6 +49,20 @@ public function respond(RequestContext $context, OctaneResponse $octaneResponse)
             );
         }
 
+        if (
+            ($octaneResponse->response instanceof StreamedResponse) &&
+            ($responseCallback = $octaneResponse->response->getCallback()) &&
+            ((new ReflectionFunction($responseCallback))->getReturnType()?->getName() === Generator::class)
```

Comment:
This could be:
```php
((new ReflectionFunction($responseCallback))->isGenerator()
```
a typehint would not be needed to know if it's a generator or not